scripts/value_sets/terminology_valueset_sync.py

```diff
@@ -0,0 +1,164 @@
+import csv
+import os
+
+import requests
+
+# Set Terminology URLS
+LOINC_BASE_URL = "https://loinc.regenstrief.org/searchapi/loincs?"
+LOINC_LAB_ORDER_SUFFIX = "query=orderobs:Order+OR+orderobs:Both&count=500"
+LOINC_LAB_RESULT_SUFFIX = "query=orderobs:Observation+OR+orderobs:Both&count=500"
+HL7_LAB_INTERP_URL = (
+    "https://www.fhir.org/guides/stats2/valueset-us.nlm.vsac-2.16.840.1.113883.1.11.78.json"
+)
+UMLS_SNOMED_LAB_VALUES_URL = (
+    "https://uts-ws.nlm.nih.gov/rest/content/current/source/SNOMEDCT_US/362981000/descendants"
+)
+
+# Get Terminology Usernames and Passwords
+LOINC_USERNAME = os.environ.get("LOINC_USERNAME")
```

Comment:
I think we should document these in the toplevel docstring, help text if we go that route and raise an appropriate error if they are undefined.